cr8/cli.py

```diff
@@ -54,15 +70,16 @@ def dicts_from_lines(lines):
         ]
     """
     lines = iter(lines)
+    parse = _create_parse()
     for line in lines:
         line = line.strip()
         if not line:
             continue  # skip empty lines
         try:
-            yield json.loads(line, object_pairs_hook=OrderedDict)
+            yield parse(line)
         except json.decoder.JSONDecodeError:
```

Comment:
To be best compatible with both `json` and `simdjson` I'd suggest catching`ValueError` here instead. `JSONDecodeError` is a subclass of ValueError that adds some extra information simdjson doesn't have yet, so we just return a ValueError.